manifests/config.pp

```diff
@@ -35,7 +35,10 @@
     })
   }
 
-  ensure_resource('file', $::jenkins::localstatedir, $dir_params)
+  if $::jenkins::manage_localstatedir {
+    ensure_resource('file', $::jenkins::localstatedir, $dir_params)
+  }
+
```

Comment:
@jniesen A similar PR, #405, disabled all of the directory resources.  Do you have any opinion as to which way to go?